lib/Dialect/LoopSchedule/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+set(LLVM_OPTIONAL_SOURCES
+  DialectRegistration.cpp
+  mlir_std_runner.cpp
+  Ops.cpp
+  )
+
+add_circt_dialect_library(CIRCTLoopSchedule
+  LoopScheduleOps.cpp
+
+  ADDITIONAL_HEADER_DIRS
+  ${PROJECT_SOURCE_DIR}/include
+  ${PROJECT_BINARY_DIR}/include
+
+  LINK_LIBS PUBLIC
+  CIRCTESI
```

Comment:
Does this dialect need a dependency on ESI?

Reply:
No, this is a mistake. I will fix it.